### PR TITLE
Scripts: Add script to publish helm charts

### PR DESCRIPTION
**Pull request by mook-as**
This adds a script to create a PR for publishing helm charts to the prod repository.  It auto-detects which sidecar (mysql, postgres, &c.) is being published and puts it in the correct chart directory.

### CODE REVIEW

**Comment by mook-as** [comment]
To test this script, fork [SUSE/kubernetes-charts-suse-com](https://github.com/SUSE/kubernetes-charts-suse-com) (or just create an unrelated one in your account), and run the new script with `GITHUB_ORGANIZATION` set to your GitHub user name.  This will make a PR into the not-real repo, and you can check the results for correctness.  Remember to set the three variables near the top of the script that it checks for.

File: scripts/create_helm_charts_pr.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  exit 1
+fi
+
+if [ -z "$GITHUB_PASSWORD" ]; then
+  echo "GITHUB_PASSWORD environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${CAP_BUNDLE}" ]]; then
+  echo "CAP_BUNDLE not set"
```
Comment (mook-as) [inline scripts/create_helm_charts_pr.sh]: We should probably rename this, because it isn't `CAP`.

Comment (andreas-kupries) [reply scripts/create_helm_charts_pr.sh]: +1

Comment (mook-as) [reply scripts/create_helm_charts_pr.sh]: Fixed in 57ffa5d2169a8e3ffa03b9a4923c641c6e18ef08

**Pull request merged**